Animations/Zoom/Right.js

```diff
@@ -26,5 +28,5 @@ var ZoomRight = {
     'transform': 'scale(.1) translateX(20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomRight', ZoomRight);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomRight', ZoomRight);
```

Comment:
This does not need to have `'ZoomRight'` passed into it since we have defined a `className` property in `ZoomRight`.